README.md

```diff
@@ -90,7 +97,7 @@ To build the typical way:
 
 Or, if you want to compile manually:
 
-1.  Execute CC=gcc -m32 CXX=g++ -m32 mkdir build && cd build && cmake .. && make
+1.  Execute CC=gcc -m64 CXX=g++ -m64 mkdir build-nwnx && cd build-nwnx && cmake .. && make
```

Comment:
`-m64` is not necessary when building on 64bit machines, and thus setting `CC` and `CXX` is also not. So just `mkdir && cd && cmake && make`